test/hisi_zip_test/testsuit.c

```diff
@@ -899,6 +901,14 @@ int test_hw(struct test_options *opts, char *model)
 	if (ret < 0)
 		goto out;
 
+	if (opts->use_env) {
+		ret = wd_comp_get_env_param(0, 0, 0, &num, &enable);
```

Comment:
I'm curious on this API. It seems that ctx & enable are not used at all. If so, why do we need to add these two parameters in this API?

Do you have any plan to use these two parameters?

By the way, you force the numa, mode and type parameter in test case. At least, mode and type are depend on cases.

Reply:
They are designed to save ctx num and flag,it is a dfx function to user.

I have changed  mode and type parameter to opts op_type and sync_mode now.

Reply:
I discuss with tanghui, it seems ctx num and enable flag is needed by openssl engine async mode.